src/app/routes/api/v3/admin/forms/admin-forms.settings.routes.ts

```diff
@@ -52,3 +55,19 @@ AdminFormsSettingsRouter.route('/:formId([a-fA-F0-9]{24})/settings').patch(
   updateSettingsValidator,
   handleUpdateSettings,
 )
+
+/**
+ * Retrieve the settings of the specified form
+ * @route GET /admin/forms/:formId/settings
+ * @group admin
+ * @produces application/json
+ * @returns 200 with latest form settings on successful update
+ * @returns 401 when current user is not logged in
+ * @returns 403 when current user does not have permissions to obtain form settings
+ * @returns 404 when form to retrieve settings for cannot be found
+ * @returns 409 when saving form settings incurs a conflict in the database
+ * @returns 500 when database error occurs
+ */
+AdminFormsSettingsRouter.route('/:formId([a-fA-F0-9]{24})/settings').get(
+  handleGetSettings,
+)
```

Comment:
sorry last comment - since this and the above endpoint are the same, can we just chain the `patch` and `get` so it's clear that they're different methods on the same endpoint?

Reply:
... How would the jsDoc work though

Reply:
can search for `AdminFormsRouter.route('/:formId([a-fA-F0-9]{24})/adminform')` to see an example

Reply:
idk if there's a best practice for this though - I find the chaining to be more clear, but if the folklore says otherwise then it's fine